scripts/stack_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+INSTANCE_HOST_DEPLOY=https://whoami.im.dev.test.c.dhis2.org
+GROUP=whoami
+
+# Whoami
+INSTANCE_NAME=whoami-test
+./deploy-whoami.sh $GROUP $INSTANCE_NAME
+sleep 3
+kubectl wait --for=condition=available --timeout=30s --namespace $GROUP deployment/$INSTANCE_NAME-whoami-go
+http --check-status "$INSTANCE_HOST_DEPLOY/$INSTANCE_NAME"
+./destroy.sh $GROUP $INSTANCE_NAME
+
+# Monolith
+INSTANCE_NAME=monolith-test
+./deploy-dhis2.sh $GROUP $INSTANCE_NAME
+sleep 3
+kubectl wait --for=condition=available --timeout=180s --namespace $GROUP deployment/$INSTANCE_NAME-core
+http --check-status --follow "$INSTANCE_HOST_DEPLOY/$INSTANCE_NAME"
+./destroy.sh $GROUP $INSTANCE_NAME
+kubectl delete pvc --namespace $GROUP data-$INSTANCE_NAME-database-postgresql-0
+
+# Database and core
+INSTANCE_NAME=db-and-core-test
+./deploy-dhis2-db.sh $GROUP $INSTANCE_NAME
+./deploy-dhis2-core.sh $GROUP $INSTANCE_NAME $INSTANCE_NAME-core
+sleep 3
+kubectl rollout status --watch --timeout=300s --namespace $GROUP statefulset/$INSTANCE_NAME-database-postgresql
+kubectl wait --for=condition=available --timeout=300s --namespace $GROUP deployment/$INSTANCE_NAME-core
+sleep 3
+http --check-status --follow "$INSTANCE_HOST_DEPLOY/$INSTANCE_NAME-core"
+./destroy.sh $GROUP "$INSTANCE_NAME-core"
```

Comment:
Having one script which runs tests is nice as we can also run it locally 😄 Downside is that if there is some unforeseen error that makes the script exit before resources are cleaned up that we will have to clean them up ourselves. Cleaning up in a job/step that runs always https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions is probably more resilient.

Reply:
But if there's an error we might want to take a look at resources which failed so I'm not sure we want to automatically want to remove them.

Reply:
Yeah I think what @tonsV2 is saying makes sense. If we just go ahead and clean up after something fails, then we would have to reproduce it to figure out the issue (unless its evident from the logs). 🤔